ts/Extensions/ScrollablePlotArea.ts

```diff
@@ -355,14 +366,14 @@ Chart.prototype.applyFixed = function (): void {
                 position: 'absolute',
                 overflow: 'hidden',
                 pointerEvents: 'none',
-                zIndex: 2
+                zIndex: 2,
+                top: 0
             },
             null as any,
             true
         );
-        this.renderTo.insertBefore(
-            this.fixedDiv,
-            this.renderTo.firstChild
+        this.scrollingContainer?.parentNode.appendChild(
+            this.fixedDiv
```

Comment:
This puts the fixed container after the scrolling container in the DOM, while it used to be before. Assuming this is not an issue?

Reply:
Does not seem to be an issue, but I'll restore the order.